README.md

```diff
@@ -1 +1,24 @@
-# Hello1-Microverse
\ No newline at end of file
+
+project: # Hello1-Microverse
+This is my first project at Microverse.
+## Built With
+.HTML
+.CSS
+## Getting Started
+to implement and run this project you should have the following prerequisites  
+## Prerequisites
+Text editor
+browser like chrome
+## Author
+ [Github](https://github.com/solog0039)
+[linkedin](www.linkedin.com/in/solomon-kidanu-62a994232)
+[tiwtter](https://twitter.com/Solomon57320119)
+
+## Contributing
+Contributions, issues, and feature requests are welcome!
+## Show your support
+give tumps if you like this project.
+## Acknowledgments
+Microverse
+## License
+This project is MIT License licensed.
```

Comment:
I can see you made a great effort creating your readme👍, to make our projects look more professional, It is a requirement that we make use of the [Microverse readme template](https://github.com/microverseinc/readme-template/blob/master/README.md)😉 kindly make use the given [Microverse template](https://github.com/microverseinc/readme-template/blob/master/README.md) to create your readme.